roles/lib_openshift/src/doc/secret

```diff
@@ -0,0 +1,115 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oc_secret
+short_description: Module to manage openshift secrets
+description:
+  - Manage openshift secrets programmatically.
+options:
+  state:
+    description:
+    - Currently present is only supported state.
+    required: true
+    default: present
+    choices: ["present", "absent", "list"]
+    aliases: []
+  kubeconfig:
+    description:
+    - The path for the kubeconfig file to use for authentication
+    required: false
+    default: /etc/origin/master/admin.kubeconfig
+    aliases: []
+  debug:
+    description:
+    - Turn on debug output.
+    required: false
+    default: False
+    aliases: []
+  name:
+    description:
+    - Name of the object that is being queried.
+    required: false
+    default: None
+    aliases: []
+  namespace:
+    description:
+    - The namespace where the object lives.
+    required: false
+    default: str
+    aliases: []
+  files:
+    description:
+    - A list of files provided for secrets
+    required: false
+    default: None
+    aliases: []
+  delete_after:
+    description:
+    - Whether or not to delete the files after processing them.
+    required: false
+    default: false
+    aliases: []
+  contents:
+    description:
+    - Content of the secrets
+    required: false
+    default: None
+    aliases: []
+  force:
+    description:
+    - Whether or not to force the operation
+    required: false
+    default: false
+    aliases: []
+  decode:
+    description:
+    - base64 decode the object
+    required: false
+    default: false
+    aliases: []
+author:
+- "Kenny Woodson <kwoodson@redhat.com>"
+extends_documentation_fragment: []
+'''
+
+EXAMPLES = '''
```

Comment:
More comments on the `state` parameter. Is is defined as `required: true` in `DOCUMENTATION` but only one of the `EXAMPLES` below shows it being set (*get ca from hawkular*) and in that example it is set to `list`. This conflicts with the documentation stating that "Currently `present` is only supported state."